packages/agents/chain-abstraction/src/libs/shared/quote.ts

```diff
@@ -82,3 +88,102 @@ export const getBridgeAmountOut = async (
 
   return amountOut;
 };
+
+export const getEstimateAmountRecieved = async (args: EstimateQuoteAmountArgs): Promise<string> => {
```

Comment:
nit: typo, should be "getEstimateAmountRecieved" and need to chnage in other files